tests/sentry/incidents/action_handlers/test_discord.py

```diff
@@ -123,7 +123,10 @@ def test_metric_alert_halt_for_rate_limited(self, mock_record_event, mock_send_m
 
     @patch(
         "sentry.integrations.discord.client.DiscordClient.send_message",
-        side_effect=ApiError(code=50001, text="Missing access"),
+        side_effect=ApiError(
+            code=403,
+            text='{"message": "Missing access", "code": 50001}',
```

Comment:
if anyone was curious about the code like me: https://discord.com/developers/docs/topics/opcodes-and-status-codes#json-json-error-codes